src/content/en/2021/performance.md

```diff
@@ -1,20 +1,532 @@
+
+Luckily, only 0.4% of sites used the async decode directive for their LCP image. And, the negative impact of asynchronous decode is not as high as lazy loading.
+
+{{ figure_markup(
+  content="354",
+  caption="Websites attempted to use native lazy-loading on LCP elements that are not images or iframes",
+  classes="big-number",
+  sheets_gid="1423728540",
+  sql_file="lcp_element_data.sql"
+  )
+}}
+
+Interestingly, 354 origins on desktop attempted to use native-lazy loading on HTML elements that do not support the loading attribute (e.g., `<div>`). The loading attribute is only supported on `<img>` and  sometimes `<iframe>` elements (see <a hreflang="en" href="https://caniuse.com/loading-lazy-attr">caniuse</a>).
+
+### Cumulative Layout Shift (CLS)
+
+{{ figure_markup(
+  image="performance-CLS-by-device.png",
+  caption="CLS performance by device",
+  description="Stacked bar chart showing CLS performance for desktop and mobile devices. For desktop devices, 62% of websites had good (< 0.1), 23% needs improvement, and 15% poor (>= 0.25) CLS performance. For mobile, 62% were good, 21% needs improvement, and 17% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=160840238&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_device.sql"
+  )
+}}
+
+<a hreflang="en" href="https://web.dev/cls/">Cumulative Layout Shift (CLS)</a> is characterized by how much shift a user experiences, not how long it takes to visually see something like FCP and LCP. As such, performance by device was fairly equivalent.
+
+{{ figure_markup(
+  image="performance-CLS-by-ect.png",
+  caption="CLS performance by connection type",
+  description="Stacked bar chart showing CLS performance for effective connection type. For offline websites, 87% had good and 4% poor performance. For slow 2G and 2G, 53-52% were good, 15% needs improvement, and 32-33% poor. For 3G, 58% were good, 16% needs improvement, and 26% poor. For 4G, 69% were good, 13% needs improvement, and 18% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1650043738&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_eff_connection_type.sql"
+  )
+}}
+
+Performance degradation from 4G to 3G and below was not as pronounced as with FCP and LCP. Some degradation exists, but it's not reflected in the device data, only the connection type.
```

Comment:
> Performance degradation from 4G to 3G and below

to

> Performance degradation from `4G` to `3G and below`

I get what you're writing here, it's just not enough visual cues if I was skimming.  Just trying to give you a clear idea of what I'm looking for.  Maybe italics would be more subtle and consistent but still clear?

Reply:
I think italics is for emphasis which would seem out of context here. You're just wanting clarification. Not sure I can think of a way at the moment so leaving this up.

Reply:
Is it:
> "4G" to "3G" and below
Or is it:
> "4G" to "3G and below"

I presume it's the former as there are 2G buckets as well, in which case not sure it's necessary to give it a different style? If it's the latter, then I agree we should make it obvious "3G and below" is one category with either code formatting or quotes.

We generally use italics to introduce new terms rather than for emphasis. As this is not the first introduction of the 4G term so, by that, italics wouldn't be appropriate here. However, it _can_ be used for emphasis but IMHO most writers overuse italics and bold for emphasis where it's not needed, which is why we discourage its use in our wikis. But there's always exceptions, and maybe this is one?

Reply:
It's both? Basically 4G is faster and everything else no matter how you slice it is slower, except offline of course.

Should I remove the other italics then? Didn't realize this was in the style guide.

Reply:
It's in the Editor's Guide: https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide

> Try to avoid over-use of bold and even italics – Call out figures should be used for important stats, and italics should be used for introducing terms.

If https://github.com/HTTPArchive/almanac.httparchive.org/commit/a8f68a3559340fc9e163ef0e7aff565e1f030d3f is all the italics you had that was fine and feel free to put back.

It's _people_ who, seemingly _randomly_ decide to use _italics_, for many _words_ mid-sentence, apparently for _emphasis_, that can be _distracting_ that we discourage. Smashing Magazine used this style (though with __bold__) but I don't particularly like it to be honest.

The number of arguments I've had with authors on this point over the years! 😀